variables.tf

```diff
@@ -38,6 +38,12 @@ DESCRIPTION
   }
 }
 
+variable "private_link_excluded_zones" {
+  type        = set(string)
+  default     = []
+  description = "A list of Private Link Private DNS Zones to exclude. The DNS zone names must match what is provided as the default values or any input to the private_link_private_dns_zones parameter e.g. privatelink.api.azureml.ms or privatelink.{regionCode}.backup.windowsazure.com or privatelink.{regionName}.azmk8s.io."
```

Comment:
Can we update this to say it also supports the map keys and add an example to the description?

Also then re run the avm pre-commit and pr-check jobs, then commit and push 👍